### ISSUE TITLE
Unable to import SQL Database instance 

### ISSUE THREAD

**Issue by ayushmankoul** [terraform-google-modules/terraform-google-sql-db#257]
### TL;DR

Idea is to import Cloud SQL instance. I have created local configuration and which is trying to plan 5 resources. I am trying to import resources:
1. module.sql-db.google_sql_database.default[0]
2. module.sql-db.google_sql_database_instance.default
3. module.sql-db.google_sql_user.default[0]
4. module.sql-db.null_resource.module_depends_on 
5. module.sql-db.random_id.user-password
When I am trying to import first resource using following command:
>terraform import module.sql-db.google_sql_database.default[0] prj-dev-psi/mcdefrand011

It throws following error :

`module.sql-db.google_sql_database.default[0]: Refreshing state... [id=projects/prj-dev-psi/instances/europe-west3/databases/mcdefrand011]
│ Error: Cannot import non-existent remote object
│
│ While attempting to import an existing object to "module.sql-db.google_sql_database.default[0]", the provider detected that  
│ no object exists with the given id. Only pre-existing objects can be imported; check that the id is correct and that it is   
│ associated with the provider's configured region or endpoint, or use "terraform apply" to create a new remote object for     
│ this resource.`

 Also,there are no import commands available for last 2 resources in documentation.

### Expected behavior

It should import SQL instance. 

### Terraform Configuration

```hcl
module "sql-db" {

  source  = "C:/sandboxrepo/modules/mysql"
  name = var.name
  project_id = var.project_id
  region = var.region
  zone = var.zone
  
  database_version = "SQLSERVER_2019_STANDARD"
  
  
  ip_configuration = {
    authorized_networks = []
    ipv4_enabled = true
    private_network = "projects/prj-man/global/networks/shared-base"
    require_ssl = true
  }  
 
  
}
```

### Terraform Version

```sh
terraform {
  required_providers {
    google = {
      source = "hashicorp/google"
      version = "3.86.0"
    }
    google-beta = {
      source  = "hashicorp/google-beta"
      version = "~> 3.45"
    }
  }
}
```

**Comment by bharathkkb** [comment]
Hi @ayushmankoul 

Is `projects/prj-dev-psi/instances/europe-west3/databases/mcdefrand011` the correct resource id - is your instance name "europe-west3"? Could you try importing with the full resource link like `terraform import module.sql-db.google_sql_database.default[0] projects/{{project}}/instances/{{instance}}/databases/{{name}}`. 

> Also,there are no import commands available for last 2 resources in documentation.

You can skip importing the null_resource and for `module.sql-db.random_id.user-password` you can skip this by providing your current `user_password` as an input.

**Comment by ayushmankoul** [comment]
> Hi @ayushmankoul
> 
> Is `projects/prj-dev-psi/instances/europe-west3/databases/mcdefrand011` the correct resource id - is your instance name "europe-west3"? Could you try importing with the full resource link like `terraform import module.sql-db.google_sql_database.default[0] projects/{{project}}/instances/{{instance}}/databases/{{name}}`.
> 
> > Also,there are no import commands available for last 2 resources in documentation.
> 
> You can skip importing the null_resource and for `module.sql-db.random_id.user-password` you can skip this by providing your current `user_password` as an input.
Thank You @bharathkkb for your response. I have resolved this issue. We are good to close this.
